probe_scraper/probe_expiry_alert.py

```diff
@@ -120,20 +129,37 @@ def parse_args():
 
 
 def main(current_date, dryrun):
-    probe_info = requests.get(PROBE_INFO_BASE_URL + "firefox/all/main/all_probes").json()
-
     current_version = get_latest_nightly_version()
     next_version = str(int(current_version) + 1)
 
-    expired_probes = find_expiring_probes(probe_info, current_version)
-    expiring_probes = find_expiring_probes(probe_info, next_version)
+    events_file_path = os.path.join(tempfile.gettempdir(), "Events.yaml")
+    download_file(BASE_URI + EVENTS_FILE, events_file_path)
+    events = EventsParser().parse([os.path.join(tempfile.gettempdir(), events_file_path)])
```

Comment:
this should reuse `events_file_path`
```suggestion
    events = EventsParser().parse([events_file_path])
```